.drone.yml

```diff
@@ -31,6 +31,22 @@ pipeline:
     commands:
       - antora generate --cache-dir cache/ --redirect-facility netlify site.yml
 
+  serve-docs:
+    image: nginx:1.15.6-alpine
+    pull: true
+    volumes:
+    - name: docker
+      path: /var/run/docker.sock
+
+  build-docs-with-search-index:
+    image: antora/antora:1.0.1
+    pull: true
+    environment:
+        DOCSEARCH_ENABLED: true 
+        DOCSEARCH_ENGINE: lunr
+    commands:
+      - antora generate --generator=./generate-site.js --cache-dir cache/ --ui-bundle-url ui-bundle.zip --redirect-facility netlify site.yml
```

Comment:
Either:
``--ui-bundle-url ui-bundle.zip`` --> ``--ui-bundle-url https://minio.owncloud.com/documentation/ui-bundle.zip``

Or:
remove at all because already defined in ``site.yml``

Reply:
Thanks for spotting that. Not sure how that slipped in.